### PR TITLE
remove Cherenkov photons from MC stack

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

File: Detectors/FIT/FT0/simulation/src/Detector.cxx [hunk 1]
```diff
@@ -358,8 +358,6 @@ Bool_t Detector::ProcessHits(FairVolume* v)
 o2::ft0::HitType* Detector::AddHit(float x, float y, float z, float time, float energy, Int_t trackId, Int_t detId)
 {
   mHits->emplace_back(x, y, z, time, energy, trackId, detId);
-  auto stack = (o2::data::Stack*)fMC->GetStack();
-  stack->addHit(GetDetId());
```
Comment (sawenzel) [inline Detectors/FIT/FT0/simulation/src/Detector.cxx:362]: this means you not adding **any** hit leaving particle to the stack. Is that ok for you?

Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/simulation/src/Detector.cxx:362]: No, it was just panic solution

Comment (sawenzel) [reply Detectors/FIT/FT0/simulation/src/Detector.cxx:362]: There is no need to panic... For the moment this is not a critical issue and we can work out a proper solution.

**Comment by AllaMaevskaya** [comment]
Its temporary solution before parameterization of photons will be ready

**Comment by ktf** [comment]
@AllaMaevskaya Any news here? 

@sawenzel if you expect changes could you review it as "Changes required" rather than "Comment" so that it does not show up in the list?

**Comment by sawenzel** [comment]
As this is not blocking anyone, we should in my opinion not merge this temporary PR but work out a proper solution.

At the moment you will reference trackIDs in the hits which do not exist in the kinematics... which will make the whole MC labelling incorrect/inconsistent.
I would suggest to close this PR and instead go on with a consistent implementation:
*) account hits to some sort of "first impeding particle / primary / mother particle / shower generating particle" with a given trackID  --> this sort of "shunting" is done by other detectors such as PHOS, ZDC, EMC
*) only save these important particles in the kinematics

**Pull request closed without merging**